functions.py

```diff
@@ -0,0 +1,20 @@
+# Modify this function to return a list of strings as defined above
+def list_benefits() -> list:
+    benefits = [
+        "More organized code",
+        "More readable code",
+        "Easier code reuse",
+        "Allowing programmers to share and connect code together"
+    ]
+    return benefits
+
+# Modify this function to concatenate to each benefit - " is a benefit of functions!"
+def build_sentence(benefit: str) -> str:
+    return benefit + " is a benefit of functions!"
+
+def name_the_benefits_of_functions():
```

Comment:
```suggestion
def name_the_benefits_of_functions() -> None:
```
you can specify when a function does not return a value using type hints [more info](https://peps.python.org/pep-0484/#using-none)

Reply:
Done